activiti-cloud-audit-service/activiti-cloud-services-audit-model/src/test/java/org/activiti/cloud/services/audit/jpa/converters/ProcessDeletedEventConverterTest.java

```diff
@@ -97,9 +90,10 @@ public void createAPIEventShouldReturnEvent() {
         assertThat(apiEvent.getProcessInstanceId()).isEqualTo(processInstanceId);
     }
 
-    private CloudRuntimeEvent<?, ?> buildEvent(String processInstanceId) {
+    private CloudRuntimeEvent<?, ?> buildEvent(String processInstanceId, String processDefinitionId) {
         ProcessInstanceImpl instance = new ProcessInstanceImpl();
         instance.setId(processInstanceId);
+        instance.setProcessDefinitionId(processDefinitionId);
```

Comment:
This is OK, because `SendDeleteCloudProcessInstanceEventCmd` will use `ProcessEngineEventsAggregator` to append process definition attributes from the execution context available inside transaction.